### PR TITLE
Changes in describe and del commands

### PR DESCRIPTION
**Pull request by SrinivasChilveri**
Fixes: #32

Fix: 
1)  from `genectl describe  <resource name>`
to  `genectl describe <resource type> <resource name>`

2)  from `genectl del  <resource name>`
to  `genectl delete  <resource type> <resource name>`

### CODE REVIEW

File: cmd/genectl/commands/delete_execution.go [hunk 1]
```diff
@@ -49,7 +49,10 @@ func NewDeleteCommand() *cobra.Command {
 }
 
 func DeleteWorkflow(cmd *cobra.Command, args []string, delExecutionFlags *delExecutionFlags) {
-	executionName := args[0]
+	if args[0] != "execution" {
```
Comment (hzxuzhonghu) [inline cmd/genectl/commands/delete_execution.go]: also support plural style

File: cmd/genectl/commands/delete_execution.go [hunk 2]
```diff
@@ -34,9 +34,9 @@ func NewDeleteCommand() *cobra.Command {
 	var delExecutionFlags delExecutionFlags
 
 	var command = &cobra.Command{
-		Use:     "del NAME [flags]",
+		Use:     "del execution NAME [flags]",
```
Comment (hzxuzhonghu) [inline cmd/genectl/commands/delete_execution.go]: change to delete, I the `del` not consistent with k8s community

File: cmd/genectl/commands/describe_execution.go [hunk 3]
```diff
@@ -53,7 +56,10 @@ func NewDescribeExecutionCommand() *cobra.Command {
 }
 
 func DescribeWorkflow(cmd *cobra.Command, args []string, describeFlags *describeFlags) {
-	executionName := args[0]
+	if args[0] != "execution" {
```
Comment (hzxuzhonghu) [inline cmd/genectl/commands/describe_execution.go]: same here

**Comment by kevin-wangzefeng** [comment]
The pull request description looks unclear, can you revise it?

**Pull request merged**